selfdrive/car/hyundai/values.py

```diff
@@ -776,22 +776,23 @@ class Buttons:
   },
   CAR.ELANTRA_HEV_2021: {
     (Ecu.fwdCamera, 0x7c4, None) : [
-      b'\xf1\000CN7HMFC  AT USA LHD 1.00 1.03 99210-AA000 200819'
+      b'\xf1\000CN7HMFC  AT USA LHD 1.00 1.03 99210-AA000 200819',
     ],
     (Ecu.fwdRadar, 0x7d0, None) : [
-      b'\xf1\000CNhe SCC FHCUP      1.00 1.01 99110-BY000         '
+      b'\xf1\000CNhe SCC FHCUP      1.00 1.01 99110-BY000         ',
+      b'\xf1\x8799110BY000\xf1\x00CNhe SCC FHCUP      1.00 1.01 99110-BY000         ',
```

Comment:
This was the only new version. Not unexpected that we now get a longer response after the fixes to fingerprinting.